### ISSUE TITLE
Support for the new LilyGo T-Display-S3

### ISSUE THREAD

**Issue by hapklaar** [kike-canaries/canairio_firmware#240]
## Summary

This [new device](https://github.com/Xinyuan-LilyGO/T-Display-S3) has a nice large screen and would be great for this project. Would you consider adding support for it?

**Comment by hpsaturn** [comment]
Yes, I already successful test of CanAirIO Sensorslib here, in this branch of this example. But it only run in serial terminal for now. But of course the idea is try to have this with the whole firmware. 

https://github.com/hpsaturn/esp32-s3-clock/tree/av/canairio_sensors_lib_test

Of course any help with this is welcome.

**Comment by hapklaar** [comment]
Just checking in, were you able to make some progress with this board?
